src/test/java/Task20/JsonParserTests/JsonParserReadTests.java

```diff
@@ -0,0 +1,53 @@
+package Task20.JsonParserTests;
+
+import com.google.gson.JsonSyntaxException;
+import lombok.extern.log4j.Log4j2;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import parser.Parser;
+import shop.Cart;
+import java.io.File;
+
+@Log4j2
+public class JsonParserReadTests {
+    private static final String  _testDataRoot = "./src/test/java/Task20/TestData/";
+
+    @DataProvider(name = "correctFiles")
+    public Object[][] correctFilesData(){
+        return new Object[][] {{ _testDataRoot + "sample-cart.json" }};
```

Comment:
according to Java naming convention fields or variables shouldn't start from underscore